server/src/routes/posts.ts

```diff
@@ -16,13 +16,20 @@ postRouter.get(
   query("page").default(1).isInt({ min: 1 }),
   query("sort")
     .default("new")
```

Comment:
I know this was here before, but since we are implementing this default functionality it would probably make sense to change the default to be the new "default" option.
```suggestion
    .default("default")
```